doc/man7/ossl-guide-libcrypto-introduction.pod

```diff
@@ -0,0 +1,398 @@
+=pod
+
+=head1 NAME
+
+ossl-guide-libcrypto-introduction, crypto
+- OpenSSL Guide: An introduction to libcrypto
+
+
+=head1 INTRODUCTION
+
+The OpenSSL cryptography library (C<libcrypto>) enables access to a wide range
+of cryptographic algorithms used in various Internet standards. The services
+provided by this library are used by the OpenSSL implementations of TLS and
+CMS, and they have also been used to implement many other third party products
+and protocols.
+
+The functionality includes symmetric encryption, public key cryptography, key
+agreement, certificate handling, cryptographic hash functions, cryptographic
+pseudo-random number generators, message authentication codes (MACs), key
+derivation functions (KDFs), and various utilities.
+
+=head2 Algorithms
+
+Cryptographic primitives such as the SHA256 digest, or AES encryption are
+referred to in OpenSSL as "algorithms". Each algorithm may have multiple
+implementations available for use. For example the RSA algorithm is available as
+a "default" implementation suitable for general use, and a "fips" implementation
+which has been validated to FIPS standards for situations where that is
```

Comment:
FIPS **140** standards.

There are lots of FIPS standards but the only one we're interested in is 140.